### PR TITLE
Secure mode JS

### PR DESCRIPTION
**Pull request by wallin**

### CODE REVIEW

File: lib/castle/middleware/sensor.rb [hunk 1]
```diff
@@ -103,7 +101,13 @@ def app_id_js_tag(env)
       def identify_js_tag(env)
```
Comment (brissmyr) [inline lib/castle/middleware/sensor.rb]: Merge `app_id_js_tag` and `identify_js_tag` and call it something like `config_js_tag`

Comment (wallin) [reply lib/castle/middleware/sensor.rb]: Yeah agree, I also want to restructure this so they end up in the same script tag

File: lib/castle/middleware/sensor.rb [hunk 2]
```diff
@@ -86,24 +94,41 @@ def find_end_of_head_open(body)
...
       end
 
-      def app_id_js_tag(env)
-        script_tag("_castle('setAppId', '#{Castle::Middleware.configuration.app_id}');", env)
+      def complete_js_content(env)
+        app_id!(env)
+        identify!(env)
+        secure!(env)
+
+        commands = ["\n", @script_commands.values.join, "\n"].join
```
Comment (brissmyr) [inline lib/castle/middleware/sensor.rb]: Not sure I like `@script_commands` to be magically set from within the `*!` methods. Those methods should rather return the contents for testability. But don't spend the time now if you feels it's a tricky fix.

Comment (wallin) [reply lib/castle/middleware/sensor.rb]: I agree, I'll remove it. It's not necessary

**Pull request merged**